twined/schema/children_schema.json

```diff
@@ -8,15 +8,14 @@
                 "type": "string"
             },
             "id": {
-                "description": "The universally unique ID of the running child twin",
+                "description": "The universally unique ID (UUID) of the running child twin",
                 "type": "string"
             },
-            "uri_env_name": {
-                "description": "Name of the environment variable containing the URI of the twin (which may contain credentials)",
-                "type": "string",
-                "pattern": "^[A-Z]+(?:_[A-Z]+)*$"
+            "gcp_project_name": {
+                "description": "Name of the Google Cloud Platform (GCP) project the child exists in.",
```

Comment:
Need to think and talk more about this on monday.

 And it shouldn’t be provider-specific.

Maybe we need a “oneOf[object]” here allowing us to define a range of service backends: https, wss, pubsub, redis, etc. Of which the only one implemented is pubsub

In our case it’ll be an object containing

Reply:
Sounds good. I also haven't dealt with the GCP credentials yet - should they go in the `credentials` strand or be associated with each child separately? Different children could be from different organisations on GCP and so need different credentials, or they could potentially be on different providers